JetUtilities/src/L2Creator.cc

```diff
@@ -804,7 +918,7 @@ void L2Creator::makeCanvas(string makeCanvasVariable) {
                 spline_func->SetRange(bounds.first,bounds.second);
                 spline_func->SetLineColor(igraph%nperpad+1);
                 spline_func->SetLineStyle(kDotted);
-                spline_func->Draw("same");
+//              spline_func->Draw("same");
```

Comment:
Why comment out the `Draw` statement? If you don't want to draw a spline this should be controlled by logic, not by commenting out code.

Reply:
It was actually done by accident, thank you for pointing it out! I fixed it.